### ISSUE TITLE
Is there a way to change the "approved translation"?

### ISSUE THREAD

**Issue by coolvitto** [x64dbg/Translations#1]
Is there a way to change the "approved translation"?

I am currently translating Japanese. 
I would like to revise the "approved translation" because the expression is difficult to understand. 
How can I make the correction? 
Please advise.

**Comment by mrexodia** [comment]
Of course, I will make you a proofreader and you will be able to revise everything. I'm not sure if you saw my comments on Crowdin, but be careful with the format strings (`%d, %s`). Due to technical limitations these **have to** show up in the same order they are in the input string, which might make the translations a bit strange at times.

**Comment by coolvitto** [comment]
Thank you very much.
I will continue to translate with the utmost care.
